.gitignore

```diff
@@ -1 +1,2 @@
 build/
+.idea/
```

Comment:
please remove this and use a local config to exclude it (either only for this project or globally on your computer): https://help.github.com/articles/ignoring-files/
this folder is not related to the project, but to your local setup